etc/postfix/main.cf

```diff
@@ -59,3 +59,8 @@ sender_canonical_classes = envelope_sender, header_sender
 sender_canonical_maps = regexp:/etc/postfix/sender_canonical
 smtp_header_checks = pcre:/etc/postfix/second_header_checks
 smtpd_data_restrictions = check_sender_access pcre:/etc/postfix/sender_access
+
+## Receive TLS mail (587)
+smtpd_tls_security_level = may
```

Comment:
that will offer to the client a `STARTTLS` that it may or may not accept it and not use TLS encryption. Shall we consider `smtpd_tls_security_level = encrypt` to force tls to clients?

Reply:
I don't mind, the impetus for this work is to pass some checks, since we are sharing the domain with Google now from what I understand.

So as long as we offer TLS - we should be fine.

Reply:
actually we shouldn't use it:
>  by setting "smtpd_tls_security_level = encrypt". According to RFC 2487 this MUST NOT be applied in case of a publicly-referenced Postfix SMTP server. This option is off by default and should only seldom be used.

http://www.postfix.org/TLS_README.html#client_tls_may